app/proguard-rules.pro

```diff
@@ -15,3 +13,16 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+
```

Comment:
http://verybadalloc.com/android/2015/12/19/special-place-for-samsung-in-android-hell/

:dancers: